src/components/stats-plus.js

```diff
@@ -0,0 +1,130 @@
+import "./stats-plus.css";
```

Comment:
This "-plus" thing has turned into an anti-pattern IMO. We really should be contributing back to our dependencies, instead of just copy-pasting them. At the very least, we should put these changes in our fork of aframe instead of bleeding it into the client.

Reply:
Sounds good, I'll get some feedback and contribute it back or split it into its own component.